bin/pt-stalk

```diff
@@ -971,8 +971,10 @@ collect_mysql_data_one() {
       fi
    fi
 
-   $CMD_MYSQLADMIN $EXT_ARGV ext -i$OPT_SLEEP_COLLECT -c$cnt >>"$d/$p-mysqladmin" &
-   mysqladmin_pid=$!
+   if ! [[ "${OPT_SKIP_COLLECTION[@]}" =~ "mysqladmin" ]]; then
```

Comment:
Bash does not support regex matching directly against arrays using =~.

use something like this
```
should_skip=false
for item in "${OPT_SKIP_COLLECTION[@]}"; do
    if [[ "$item" == "mysqladmin" ]]; then
        should_skip=true
        break
    fi
done

if ! $should_skip; then
    "$CMD_MYSQLADMIN" $EXT_ARGV ext -i"$OPT_SLEEP_COLLECT" -c"$cnt" >> "$d/$p-mysqladmin" &
    mysqladmin_pid=$!
fi
```

Reply:
Fixed, please review again